rs/orchestrator/src/metrics.rs

```diff
@@ -92,6 +93,11 @@ impl OrchestratorMetrics {
                 "orchestrator_fstrim_duration_milliseconds",
                 "The duration of the last fstrim call, in milliseconds",
             ),
+            critical_error_task_panicked: metrics_registry.int_counter_vec(
+                "orchestrator_task_panicked",
```

Comment:
I think the [prometheus naming convention](https://prometheus.io/docs/practices/naming/) suggests adding a `total` suffix to metrics that are counters, i.e:
```suggestion
                "orchestrator_tasks_panicked_total",
```